### PR TITLE
New script - tuner

### PR DESCRIPTION
**Pull request by markwheeler**
Very simple script that displays the existing SC pitch polls.

### CODE REVIEW

File: scripts/mark_eats/tuner.lua [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+local SCREEN_FRAMERATE = 15
+local screen_dirty = true
+
+local current_freq = -1
+local last_freq = -1
+
+
+-- Encoder input
+function enc(n, delta)
+
```
Comment (pq) [inline scripts/mark_eats/tuner.lua]: maybe consider removing `enc` (or adding TODOs)?

File: scripts/mark_eats/tuner.lua [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+  if n == 2 then
+          
+  elseif n == 3 then
+    
+  end
+end
+
+-- Key input
+function key(n, z)
+  if z == 1 then
```
Comment (pq) [inline scripts/mark_eats/tuner.lua]: same here.  is the `key` function a placeholder for future stuff?

Comment (markwheeler) [reply scripts/mark_eats/tuner.lua]: Yeah I just find them useful for testing and updates but can remove if need be

Comment (pq) [reply scripts/mark_eats/tuner.lua]: not at all.  if it's not an oversight, all good!

**Pull request merged**